NitroxClient/MonoBehaviours/PlayerMovementBroadcaster.cs

```diff
@@ -11,62 +11,66 @@ namespace NitroxClient.MonoBehaviours;
 
 public class PlayerMovementBroadcaster : MonoBehaviour
 {
-    public const float BROADCAST_INTERVAL = 0.05f;
-    private LocalPlayer localPlayer;
+    /// <summary>
+    ///     Amount of physics updates to skip for sending location broadcasts.
+    ///     TODO: Allow servers to set this value for clients. With many clients connected to the server, a higher value can be preferred.
+    /// </summary>
+    public const int LOCATION_BROADCAST_TICK_SKIPS = 1;
 
-    private float time;
+    private LocalPlayer localPlayer;
+    private int locationBroadcastSkipCounter;
 
     public void Awake()
     {
         localPlayer = NitroxServiceLocator.LocateService<LocalPlayer>();
     }
 
-    public void Update()
+    public void FixedUpdate()
     {
-        time += Time.deltaTime;
-
-        // Only do on a specific cadence to avoid hammering server
-        if (time >= BROADCAST_INTERVAL)
+        // Throttle location broadcasts to not run on every physics tick.
+        if (locationBroadcastSkipCounter++ < LOCATION_BROADCAST_TICK_SKIPS)
```

Comment:
Would it potentially be safer to invert this? Just incase it goes over the location broadcast tick skips and then it will never limit it

Reply:
Like this?
```cs
// Throttle location broadcasts to not run on every physics tick.
if (--locationBroadcastSkip >= 0)
{
    return;
}
// Reset counter.
locationBroadcastSkip = LOCATION_BROADCAST_TICK_SKIPS;
```

Reply:
I do like the design you have proposed but I think doing it that way would mean the first update would be skipped with the value of locationBroadcastSkip being -1. If we increment it after it will allow it to run on the first update and then skip others until it reaches the LOCATION_BROADCAST_TICK_SKIPS. 
```
// Throttle location broadcasts to not run on every physics tick.
if (locationBroadcastSkip-- > 0)
{
    return;
}
// Reset counter.
locationBroadcastSkip = LOCATION_BROADCAST_TICK_SKIPS;
```